aips/aip-31.md

```diff
@@ -37,6 +45,10 @@ Some delegation pools may require a KYC process before accepting delegated stake
 
 Delegation pools with allowlisting will have to manage their own KYC process. 
 
+The minimum stake requirement for active and pending-inactive stake per delegator is 10 APT.
+A delegator owning exactly 10 APT active stake might not be able to unlock or be evicted by the pool owner: 10 APT could be worth 9,99..9 APT when unlocked due to rounding errors caused by the "active to pending-inactive" conversion and the unlock would revert.\
```

Comment:
Remove "\" at the end of the sentence?

Reply:
this generates a newline in generated md, is not visibile in md preview